src/parser.rs

```diff
@@ -1935,7 +1933,7 @@ impl Parser {
         let (quantity, percent) = if self.parse_one_of_keywords(&["ROW", "ROWS"]).is_some() {
             (None, false)
         } else {
-            let quantity = self.parse_sql_value()?;
+            let quantity = self.parse_value_expr()?;
```

Comment:
I had to get back to [the original PR](https://github.com/andygrove/sqlparser-rs/pull/69#issuecomment-497160698) to remember why we store quantity as an `Expr(Value())` rather than just `Value`: at least Postgres claims to support more complex expressions - while we don't parse them, we "future-proofed" the AST in case we do.

Reply:
Makes sens.